packages/shared/src/modules/courses/libs/types/course-search-request-dto.type.ts

```diff
@@ -0,0 +1,6 @@
+type CourseSearchRequestDto = {
+	search: string;
+	vendorsKeys?: string;
```

Comment:
```suggestion
	vendorsKey?: string;
```
Or if it should be an array, fix that

Reply:
Here is a problem, because on client I use URLSearchParams to make a request, but array of strings come to backend as string with commas

Reply:
@fshabanov value for this field is vendors keys separated by coma. Can I leave it as it is?

Reply:
In order to get array, we have to send `&vendorsKey[]=udemy&vendorsKey[]=coursera`. But in this case in controller I get:
`query Empty <[Object: null prototype] {}> {
  search: 'google',
  'vendorsKey[]': [ 'udemy', 'coursera' ]
}`

Reply:
[Here](https://github.com/fastify/fastify/issues/2841) they advise to use [qs lib](https://www.npmjs.com/package/qs). But I think it is not fast to implement this. So, I have just changed name to `vendorsKey`